services/catalog/src/simcore_service_catalog/api/routes/services.py

```diff
@@ -184,6 +191,109 @@ async def cached_registry_services() -> Deque[Tuple[str, str, Dict[str, Any]]]:
     return [s for s in services_details if s is not None]
 
 
+SIMCORE_SERVICE_SETTINGS_LABELS: Final[str] = "simcore.service.settings"
+
+
+@router.get(
+    "/{service_key:path}/{service_version}/resources",
+    response_model=Dict[ResourceName, ResourceValue],
+    **RESPONSE_MODEL_POLICY,
+)
+@cached(
+    ttl=DIRECTOR_CACHING_TTL,
+    key_builder=lambda f, *args, **kwargs: f"{f.__name__}_{kwargs['service_key']}_{kwargs['service_version']}",
+)
+async def get_service_resources(
+    service_key: ServiceKey,
+    service_version: ServiceVersion,
+    director_client: DirectorApi = Depends(get_director_api),
+    default_service_resources: ServiceResources = Depends(
+        get_default_service_resources
+    ),
+):
+    # TODO: --> PC: I'll need to go through that with you for function services,
```

Comment:
Yes, we have to review this. Currently these servcies do not consume any significant resources. So either we set them to 0 or remove them from the listing. Let's discuss about it offline.

Reply:
0 is out of the discussion. I disagree. or we will end up with things suddenly consuming resource somewhere totally wild.